salesagility/SuiteCRM
# Fix #6022 - API v4_1 get_module_fields method does not return parentenum

## Description

See issue #6022



## Motivation and Context

On API v4_1 `get_module_fields` method is not returning `parentenum` values.

## How To Test This

1. Call `get_module_fields` on `Cases` module.
2. See that `state` and `status` fields doesn't have `parentenum` value.
3. Apply PR and repeat steps 1 and 2.
4. See that now `parentenum` value is pressent on `state` and `status` fields.

## Types of changes


### Final checklist
